packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import get from 'lodash/get'
+import { withMutations } from 'cozy-client'
+import { withRouter } from 'react-router'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import Modal, {
+  ModalContent,
+  ModalHeader
+} from 'cozy-ui/transpiled/react/Modal'
+import Button from 'cozy-ui/transpiled/react/Button'
+import { Text } from 'cozy-ui/transpiled/react/Text'
+import { withBreakpoints } from 'cozy-ui/transpiled/react'
+import palette from 'cozy-ui/transpiled/react/palette'
+
+import accountMutations from '../connections/accounts'
+import triggersMutations from '../connections/triggers'
+import * as triggersModel from '../helpers/triggers'
+import TriggerManager from './TriggerManager'
+
+class EditAccountModal extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      trigger: null,
+      account: null,
+      fetching: true,
+      error: false
+    }
+
+    this.handleKonnectorJobSuccess = this.handleKonnectorJobSuccess.bind(this)
+  }
+
+  componentDidMount() {
+    const { accountId, accounts } = this.props
+    /**
+     * @TODO In theory we can have several trigger for the same account
+     * so this code will not work as excepted.
```

Comment:
```suggestion
     * If so this code will not work as excepted.
```

The actual problem would be having multiple triggers for the same account, which is mostly a theoretical problem at this point. The commend makes it sound like the code needs patching ASAP.